CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+- **[NEW]** `ItineraryLocation`
+- **[NEW]** `ItineraryCollapsible`
```

Comment:
Personnally, I won't export these two and keep them internal to ui-library (as we do for `Item`)

Reply:
Meaning that i'll:
- put these two components inside `src/itinerary`
- won't create a story about them. 

We should use them though `Itinerary` only, not as standalone components

Reply:
Remove internal components from changelog